### ISSUE TITLE
suggestion to make new wallet creation more user friendly

### ISSUE THREAD

**Issue by TI-ZA** [coranos/elastos-light-wallet#10]
A new user wont know how to get started here. you have to "generate a new private key"  to create a new wallet and then paste it in "enter key" section, maybe reword it as "Create New Wallet" ? 

also adding mnemonics would be great since all ELA wallets use mnemonics.

**Comment by coranos** [comment]
since this wallet is meant to be used with the ledger nano s, that part is a bit clunky, yes.

**Comment by coranos** [comment]
fixed in #552d56a82eb37e32d9a611e3b5988e2d7fb17cd7
